doc/040_backup.rst

```diff
@@ -216,7 +216,7 @@ On most Unixy shells, you can either quote or use backslashes. For example:
 By specifying the option ``--one-file-system`` you can instruct restic
 to only backup files from the file systems the initially specified files
 or directories reside on. In other words, it will prevent restic from crossing
-filesystem boundaries when performing a backup.
+filesystem boundaries, mountpoints and subvolumes when performing a backup.
```

Comment:
I'd prefer to just explicitly add `subvolumes` to the previously mentioned `filesystem`. mountpoints on their own are only a necessary, but no sufficient condition. For example bind mounts within a filesystem are crossed even with the `--one-file-system` option.